database/transaction_test.go

```diff
@@ -2,13 +2,13 @@ package database_test
 
 import (
 	"errors"
-	"testing"
 	"github.com/genjidb/genji/database"
 	"github.com/genjidb/genji/document"
 	"github.com/genjidb/genji/document/encoding/msgpack"
 	"github.com/genjidb/genji/engine/memoryengine"
 	"github.com/genjidb/genji/key"
 	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
This change shows that your editor is not well configured yet. `goimports` should have move this one with other built-in dependencies (e.g `errors`). 

`goimports` sorts the dependencies like this: 
- 1st block stdlib dependencies
- 2nd block the remaining deps

It can also separate your module dependency from the 3rd party dependencies in a 3rd block.

Reply:
Now my IDE is configured.

Reply:
@tzzed It must have been done because your editor wasn't properly configured. Now that it is properly setup, please correct this file because as you can see, it is modified for no reason and the modification is wrong because the order is not the right one.